src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -433,6 +438,48 @@ class ComposterOverlay {
         return first ?: error("First is empty!")
     }
 
+    private fun getMaterials(internalName: String, itemName: String, itemsNeeded: Double) {
```

Comment:
The method name should naybe be more desciptive. a "get" in coding represents normally grabbing data/values, not the getting the items from hypixel. maybe requestMaterials or collectMaterials?

Reply:
retrieveMaterials?